columnflow/plotting/example.py

```diff
@@ -225,6 +254,7 @@ def plot_all(
 def plot_variable_per_process(
     hists: OrderedDict,
     config_inst: od.config,
+    category_inst: od.category,
     variable_insts: list[od.variable],
     style_config: dict | None = None,
```

Comment:
The order `od` types should be all upper case (`od.Config`, `od.Category`, `od.Variable`), otherwise you are referring to a the modules (`od.config` -> `order/config.py`).